src/bin/pred.rs

```diff
@@ -223,24 +225,31 @@ impl SynthLanguage for Pred {
     }
 
     fn init_synth(synth: &mut Synthesizer<Self>) {
+        let cvec_len = 300;
```

Comment:
this is because too many unsound rules if cvec is small?

Reply:
300 was chosen semi-arbitrarily. Other domains have cvecs with hundreds to a couple-thousand elements, so 300 seemed like in the right ballpark.